assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -67,6 +81,20 @@ class TextBlockEdit extends Component {
 		if ( checkBlockDimensions ) {
 			maybeUpdateBlockDimensions( this.props );
 		}
+
+		// If the state changed to editing, focus on the text.
+		if ( this.state.isEditing && ! prevState.isEditing ) {
+			const textInput = document.querySelector( '.is-selected .wp-block-amp-amp-story-text' );
+			if ( textInput ) {
```

Comment:
Just realized that this could also be a helper function, it's a duplicate with the CTA block.